src/metabase/task/sweep_query_analysis.clj

```diff
@@ -30,7 +30,7 @@
   ([]
    (analyze-cards-without-complete-analysis! query-analysis/queue-analysis!))
   ([analyze-fn]
-   (let [cards (t2/reducible-select [:model/Card :id]
+   (let [cards (t2/reducible-select [:model/Card :id :dataset_query :entity_id :collection_id :name]
```

Comment:
Are these missing `:created_at` or does something else add it?

Reply:
If I add `:created_at` here then I no longer see the backfill warning for Card 131. I guess the NULL `collection_id` was a red herring.

Reply:
Whoops, this is my bad. I didn't get the set of fields correct.